web/src/components/atoms/UploadAsset/index.tsx

```diff
@@ -0,0 +1,16 @@
+import Button from "@reearth-cms/components/atoms/Button";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Upload from "@reearth-cms/components/atoms/Upload";
+import { UploadProps } from "antd/lib/upload/interface";
+
+const UploadAsset: React.FC<UploadProps> = ({ ...props }) => {
+  return (
+    <Upload {...props}>
+      <Button type="primary" icon={<Icon.upload />}>
+        Upload Asset
+      </Button>
+    </Upload>
+  );
+};
+
+export default UploadAsset;
```

Comment:
I feel like we don't need to make something as specific as this into an Atom. Keep in mind the idea of atoms are 1. reusability for us 2. useful for someone that might want to use it in another project.
If you just put this directly in to the molecule it is being used I think would be fine.

Reply:
I agree.